src/Processors/QueryPlan/QueryPlan.h

```diff
@@ -58,8 +58,7 @@ class QueryPlan
     void optimize(const QueryPlanOptimizationSettings & optimization_settings);
 
     QueryPipelineBuilderPtr buildQueryPipeline(
-        const QueryPlanOptimizationSettings & optimization_settings,
-        const BuildQueryPipelineSettings & build_pipeline_settings);
+        const QueryPlanOptimizationSettings & optimization_settings, const BuildQueryPipelineSettings & build_pipeline_settings);
```

Comment:
Let's avoid pointless reformatting in all files.

Reply:
Removed the linter commit. Had to force push.